DataWrite.py

```diff
@@ -133,17 +133,22 @@ def write_data(self, data_instance_array):
         # Write events
         try:
             data_to_write = []
+            redis_flag = False
             for data_instance in data_instance_array:
                 # Basic check if data have same schema
                 if data_instance["cenote"]["url"] != first_event["cenote"]["url"]:
                     raise Exception("Data don't belong to the same table!")
 
+                # Check for redisHist flag
+                if "redisHist" in data_instance:
+                    redis_flag = True
+

```

Comment:
Given https://github.com/AuthEceSoftEng/cenote-api/pull/39#discussion_r509236354, maybe this should change

Reply:
Done.